BugReporter/BugReportForm.cs

```diff
@@ -243,5 +243,8 @@ public TestAccessor(BugReportForm form)
 
             public static bool CheckContainsInfo(string input) => BugReportForm.CheckContainsInfo(input);
         }
+
+        [GeneratedRegex(@"\s|\r|\n")]
+        private static partial Regex WhitespaceRegex();
```

Comment:
Let's keep these automagically generated methods at the top of the class after fields.
Ditto other instances.

Reply:
All generated occurrences to be changed, including future conversions then.

Reply:
Changed